src/components/Sidebar/CartContainer.js

```diff
@@ -0,0 +1,11 @@
+import toJS from 'util/to-js';
+import { connect } from 'react-redux';
+import Cart from './Cart';
+import { sectionsSelector } from '../Calendar/CalendarViewContainer';
```

Comment:
Now that this selector is being used across various components (especially calendar and sidebar), we should make a `selectors` directory under `src` and throw the selector definition under an `index.js` in there. That way we can do `import { sectionsSelector } from 'selectors';`.

Reply:
This is done! I moved the test to a more appropriately named file within `selectors` as well. There are no longer any `CalendarViewContainer` tests since there doesn't remain anything to test.